packages/components/src/unit-control/types.ts

```diff
@@ -24,3 +37,20 @@ export type WPUnitControlUnit = {
 };
 
 export type WPUnitControlUnitList = Array< WPUnitControlUnit > | false;
+
+export type UnitControlLabelProps = InputControlLabelProps;
```

Comment:
We can probably avoid the re-export, and use `InputControlLabelProps` directly where needed?

Reply:
Good call, I've removed this export.